tests/pytorch/test_nn.py

```diff
@@ -1731,3 +1731,10 @@ def test_MetaPath2Vec(idtype):
     model = model.to(dev)
     embeds = model.node_embed.weight
     assert embeds.shape[0] == g.num_nodes()
+
+@pytest.mark.parametrize('max_degree', 5)
+@pytest.mark.parametrize('embedding_dim', 16)
```

Comment:
```suggestion
@pytest.mark.parametrize('max_degree', [2, 5])
@pytest.mark.parametrize('embedding_dim', [8, 16])
@pytest.mark.parametrize('direction', ['in', 'out', 'both'])
```
Add test for different directions and max_degree w/ or w/o clamping. Please modify the function below accordingly.